ButtplugServerGUI/ButtplugConfig.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ButtplugServerGUI
+{
+    public class ButtplugConfig
+    {
+        private readonly string configFile;
+        private JObject config;
+        private DateTime modtime;
+
+        public ButtplugConfig(string app)
+        {
+            configFile = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), app, "config.json");
+
+            config = new JObject();
+            modtime = DateTime.MinValue;
+            LoadConfig();
+        }
+
+        private void LoadConfig()
+        {
+            if(File.Exists(configFile) && File.GetLastWriteTimeUtc(configFile) > modtime)
+            {
+                try
+                {
+                    config = JObject.Parse(File.ReadAllText(configFile));
```

Comment:
Worth writing a schema and doing a check here? Can totally happen as a follow up, doesn't need to land in this.